### PR TITLE
PIC-4274: Adding logs and calling version2 of api for getting case list

### PR DESCRIPTION
**Pull request by sameermoj**
Changes for investigation : https://dsdmoj.atlassian.net/browse/PIC-4274

### CODE REVIEW

File: tests/services/case-service.test.js [hunk 1]
```diff
@@ -158,7 +158,7 @@ describe('Case service', () => {
   })
 
   it('should call the API to request case list data', async () => {
-    moxios.stubRequest(`${apiUrl}/court/SHF/cases?date=2020-01-01`, {
```
Comment (Obsiye) [inline tests/services/case-service.test.js:161]: Hi, just want to check if you are aware that the VERSION2 of that api paginates the results by default it returns 20 results. Is that ok?

Also the related court-case-service ticket is [here](https://dsdmoj.atlassian.net/browse/PIC-2377)

Comment (sameermoj) [reply tests/services/case-service.test.js:161]: Hey thankd @Obsiye , this was discussed with @johnyfarrar and accordingly made changes

Comment (sameermoj) [reply tests/services/case-service.test.js:161]: and yes, thats fine

**Review by Obsiye — APPROVED** [review]

**Review by johnyfarrar — APPROVED** [review]
LGTM

**Pull request merged**